ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -103,6 +103,9 @@
 /* Accessibility label for the avatar (or profile) view */
 "Accessibility.LargeTitle.ProfileView" = "Account Profile";
 
+/* Accessibility label for the navigation bar's back button */
+"Accessibility.NavigationBar.BackLabel" = "Back";
+
```

Comment:
Is this supposed to be in alphabetical order? Seems weird that `Accessibility.AvatarView.LabelFormat` is at the bottom.

Reply:
The file is mostly alphabetical but there are some strings that are out of place but I can fix em up.

Reply:
Awesome, thanks Lamine!